strongbox-user-management/src/main/java/org/carlspring/strongbox/config/UsersConfig.java

```diff
@@ -157,6 +170,34 @@ private User toInternalUser(org.carlspring.strongbox.security.User user)
         internalUser.setRoles(user.getRoles());
         internalUser.setSalt(user.getSeed() + "");
 
+        // load features
+        org.carlspring.strongbox.security.Features features = user.getFeatures();
+        if (features != null)
```

Comment:
Why do we need all these lines inside if? And what if there appears one more if inside it? 
So it will be complexity+1, and we can avoid it with above.